pkg/client/unversioned/remotecommand/remotecommand.go

```diff
@@ -194,12 +194,19 @@ func (e *Streamer) doStream() error {
 		if err != nil {
 			return err
 		}
-		defer remoteStdin.Reset()
```

Comment:
the deferred remoteStdin.Close() in the goroutine makes sense, but I would keep the deferred `Reset()` in place